### ISSUE TITLE
[PSPE] Sun movement is jerky on PE

### ISSUE THREAD

**Issue by magicus** [ProtocolSupport/ProtocolSupportPocketStuff#22]
The sun is wiggling jerkily up and down all the time on PE. My guess is that it's somehow moving to fast and needs to be readjusted back. Anyway it looks quite funny and a little disturbing. (But rather low priority, I realize...)

**Comment by 7kasper** [comment]
🤔

**Comment by 7kasper** [comment]
M8ght be related to #21 @magicus is this still an issue?

**Comment by magicus** [comment]
Yes, this is still an issue. I know understand a bit more about it. :-)

This happens when gamerule doDaylightCycle is false. Apparently the sun is still moving slightly, and then it is being reset by the server, over and over. So overall, the time does not progress, but the sun is moving all the time nevertheless. :-)

**Comment by magicus** [comment]
When doDaylightCycle is true, the movement is smooth and as expected.

**Comment by 7kasper** [comment]
Yes. I think when gamerules are implemented this will be fixed :)

**Comment by magicus** [comment]
Aha, gamerules are still missing? Sorry, I've missed that.
